src/smartpeak/source/ui/InfoWidget.cpp

```diff
@@ -62,7 +64,7 @@ namespace SmartPeak
       std::ostringstream os;
       os << "File loading failed.  Check the `Information` log.";
       ImGui::PushStyleColor(ImGuiCol_Text, ImVec4(1.0f, 0.0f, 0.0f, 1.0f));
-      ImGui::Text(os.str().c_str());
+      ImGui::Text("%s", os.str().c_str());
```

Comment:
I'm curious, why using that pattern "%s", os.str().c_str() instead of os.str().c_str() ? is it problematic?

Reply:
it is not a formatted as a string literal which makes it insecure so we're treating the string as an argument, i've changed all occurrences